src/main/java/liuni/TwitterTimeline.java

```diff
@@ -27,4 +33,37 @@ public void getTimeline() {
             System.exit(-1);
         }
     }
+
+    public JSONObject[] getTimelineJson() {
+        ConfigurationBuilder cb = new ConfigurationBuilder();
+        cb.setJSONStoreEnabled(true);
+
+        Twitter twitter = new TwitterFactory(cb.build()).getInstance();
+        try {
+            List<Status> statuses = twitter.getHomeTimeline();
+
+            JSONObject[] jsonObjects = new JSONObject[statuses.size()];
+            for (int i = 0; i < statuses.size(); i++) {
+                try {
+                    String rawJsonStr = TwitterObjectFactory.getRawJSON(statuses.get(i));
```

Comment:
i see you're getting the json just to parse it, seems a bit much.  Can all of these code be simplified?